kube/file-vault/file-vault-ingress.yml

```diff
@@ -3,13 +3,26 @@ kind: Ingress
 metadata:
   {{ if eq .KUBE_NAMESPACE .BRANCH_ENV }}
   name: file-vault-ingress-{{ .DRONE_SOURCE_BRANCH }}
+  {{ file .FILEVAULT_INGRESS_EXTERNAL_ANNOTATIONS | indent 2 }}
+  {{ else if eq .KUBE_NAMESPACE .UAT_ENV }}
+  name: file-vault-ingress
+  {{ file .FILEVAULT_INGRESS_EXTERNAL_ANNOTATIONS | indent 2 }}
   {{ else }}
   name: file-vault-ingress
+  # This is temporary change using ingress internal for prod. We will revert back the change after PECC expires
+  # https://support.acp.homeoffice.gov.uk/servicedesk/customer/portal/1/ACP-19781
+  #{{ file .FILEVAULT_INGRESS_EXTERNAL_ANNOTATIONS_PROD | indent 2 }}
+  {{ file .FILEVAULT_INGRESS_INTERNAL_ANNOTATIONS | indent 2 }}
   {{ end }}
```

Comment:
I would recommend to swap this around to avoid code duplicate 

```
{{ if eq .KUBE_NAMESPACE .PROD_ENV }}
name: file-vault-ingress
# This is temporary change using ingress internal for prod. We will revert back the change after PECC expires
# https://support.acp.homeoffice.gov.uk/servicedesk/customer/portal/1/ACP-19781
# {{ file .FILEVAULT_INGRESS_EXTERNAL_ANNOTATIONS_PROD | indent 2 }}
{{ file .FILEVAULT_INGRESS_INTERNAL_ANNOTATIONS | indent 2 }}
{{ else }}
name: file-vault-ingress-{{ .DRONE_SOURCE_BRANCH }}
{{ file .FILEVAULT_INGRESS_EXTERNAL_ANNOTATIONS | indent 2 }}
{{ end }}
```

Reply:
Thanks Darya,
The solution you suggested might not work for us. As you know Branch, UAT urls have different Ingress patterns. Branch env Ingress with the feature branch name included We can have many ingresses. But for UAT we have one Ingress url for file vault. else condition apply changes to Branch and UAT in the suggested appraoach.